### ISSUE TITLE
Revamp path selection flow

### ISSUE THREAD

**Issue by I3uckwheat** [TheOdinProject/discussions#44]
Currently there is an issue where people ask about which path they should take before they've even done the foundations course. While we do not want to hide the other paths, we should discourage people from worrying about what their decision has to be. In the Discord we have discussed a few ways of doing this, one of them being some visual changes on the "view course" page. 

![image](https://user-images.githubusercontent.com/4411701/99566643-6adc9a00-298a-11eb-8894-803ddba6797c.png)
(In this example above, the other courses are just not drawn out, but implied to look like the first box )

![image](https://user-images.githubusercontent.com/4411701/99566655-6dd78a80-298a-11eb-8bbf-859e272a8afc.png)

The two screenshots above show a method of making it a bit more difficult for a student to get lost. The idea is to have a nice big "START" button that gets disabled if they haven't finished the prerequisites. We would want to leave a "view this course anyway" link so that people are not totally blocked from viewing all the content, especially if they choose to not login or create an account.

IF this is something we want to do, here are some considerations that come along with it:

* People that refuse to create an account could have a harder time finding the path they're working on
* We may want to consider a new page/modal calling for people to sign up when they click "start" to help them track their progress
* Possibly add some content about how they will later choose a path, and be prepared to chose that path when the time comes
* Maybe show the courses as a "tree" instead of a list to help show that students do not have to take both paths
* Break out the "getting hired" section into it's own course and set the prereqs to "foundations" and "Ruby or Node". 
* prereqs could be parts of each path, and not just the entire path. People often get hired before finishing the Rails course. 

Ada could come up with some better designs for this, but I really think having a more obvious START HERE point would be super useful while still showing the students what is coming up. This also allows us to add more courses easily and keeps it sustainable. Let's hear some thoughts!

**Comment by leosoaivan** [comment]
I like giving users the ability to view later courses as they wish. I think they should also be given the ability to start the later tracks as they wish, even if we discourage it. At the end of the day, people should still have the freedom to decide for themselves.

**Comment by I3uckwheat** [comment]
@leosoaivan Which is why the "view anyway" link would be there as referenced above, I agree that is important. 

We could add a "Start this course" at the top of the view page as well to auto sign them up for it.

**Comment by KevinMulhern** [comment]
> Currently there is an issue where people ask about which path they should take before they've even done the foundations course.

Is this as big of a problem as we think it is? based on the chat it seems the number of people asking that question has went down a lot since your conclusion lesson changes went out.

**Comment by I3uckwheat** [comment]
@KevinMulhern I don't think this is an urgent concern. But if we're re-working the pages anyway, this is a nice-to-have.

After doing some watching in the chat, this seems to be a bigger issue than previously thought.
